### PR TITLE
Kondo index

### PR DESCRIPTION
**Pull request by hiroya210**
Added index page fro provider

### CODE REVIEW

**Comment by spioupiou** [comment]
I am only being picky with style lol but it works great!

Screen Shot 2022-01-27 at 21 21 09

File: app/controllers/kondos_controller.rb [hunk 1]
```diff
@@ -3,11 +3,23 @@ class KondosController < ApplicationController
...
 
   def index
-    unless params[:search_kondos] == ""
-      @kondos = @kondos.where("LOWER(prefecture) like ?","%#{params[:search_kondos].to_s.downcase}%")
-    else
-      @kondos = Kondo.all
+
+    #Renter's Page
+    if current_user.role == "renter"
+      @kondos = Kondo.order(created_at: :desc)
```
Comment (carlnoval) [inline app/controllers/kondos_controller.rb:9]: This line is very persistent, lets exterminate it.

File: app/controllers/kondos_controller.rb [hunk 2]
```diff
@@ -3,11 +3,23 @@ class KondosController < ApplicationController
   skip_before_action :authenticate_user!, only: [:index, :show]
 
   def index
-    unless params[:search_kondos] == ""
-      @kondos = @kondos.where("LOWER(prefecture) like ?","%#{params[:search_kondos].to_s.downcase}%")
-    else
-      @kondos = Kondo.all
+
+    #Renter's Page
+    if current_user.role == "renter"
```
Comment (carlnoval) [inline app/controllers/kondos_controller.rb:8]: Similar to show controller.

Nothing wrong with this too but we can now be more fancy and take advantage of enum generated methods for our user roles like so:
```ruby
if current_user.renter?
```

File: app/controllers/kondos_controller.rb [hunk 3]
```diff
@@ -3,11 +3,23 @@ class KondosController < ApplicationController
...
+
+      unless params[:search_kondos] == ""
+        @kondos = @kondos.where("LOWER(prefecture) like ?","%#{params[:search_kondos].to_s.downcase}%")
+      else
+        @kondos = Kondo.all
+      end
+      
     end
+    #Provider's Page
+    if current_user.role == "provider"
```
Comment (carlnoval) [inline app/controllers/kondos_controller.rb:19]: We can use enums here too.

File: app/models/user.rb [hunk 4]
```diff
@@ -13,10 +13,10 @@ class User < ApplicationRecord
...
-                                Hokkaido Hyogo Ibaraki Ishikawa Iwate Kagawa Kagoshima Kanagawa
-                                Kochi Kumamoto Kyoto Mie Miyagi Miyazaki Nagano Nagasaki Nara Niigata
-                                Okayama Okinawa Oita Osaka Saga Saitama Shiga Shimane Shizuoka Tochigi
-                                Tokushima Tokyo Tottori Toyama Wakayama Yamagata Yamaguchi Yamanashi], allow_nil: false }
+  # validates :prefecture,
+  #           inclusion: { in: %w[Aichi Akita Aomori Chiba Ehime Fukui Fukuoka Fukushima Gifu Gunma Hiroshima
+  #                               Hokkaido Hyogo Ibaraki Ishikawa Iwate Kagawa Kagoshima Kanagawa
+  #                               Kochi Kumamoto Kyoto Mie Miyagi Miyazaki Nagano Nagasaki Nara Niigata
+  #                               Okayama Okinawa Oita Osaka Saga Saitama Shiga Shimane Shizuoka Tochigi
+  #                               Tokushima Tokyo Tottori Toyama Wakayama Yamagata Yamaguchi Yamanashi], allow_nil: false }
```
Comment (carlnoval) [inline app/models/user.rb:21]: Yeah not sure why we're commenting these out

File: app/views/kondos/index.html.erb [hunk 5]
```diff
@@ -1,5 +1,9 @@
 <!-- Delete me later!-->
-<h1> Kondo Index Page </h1>
+<% if current_user.role == "renter" %>
```
Comment (carlnoval) [inline app/views/kondos/index.html.erb:2]: Enums :)

**Pull request closed without merging**